gt-elasticsearch/src/main/java/mil/nga/giat/data/elasticsearch/ElasticParserUtil.java

```diff
@@ -311,13 +308,14 @@ private Coordinate createCoordinate(final List<Object> posList) {
         List<Object> values = new ArrayList<>();
         if (!keys.isEmpty()) {
 
-            Object entry = source.get(keys.get(0));
+            final Object entry = source.get(keys.get(0));
 
             if (entry == null) {
-                entry = source.get(name);
+                readField(source.get(name), Collections.EMPTY_LIST, values);;
```

Comment:
The test in the original PR implementation didn't fail. Is there an test update or additional test for the issue this update fixes?